core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/endpoint/DistroXV1Endpoint.java

```diff
@@ -592,4 +592,11 @@ FlowIdentifier modifyProxyInternal(@ValidCrn(resource = {CrnResourceDescriptor.D
     @Path("rotate_secret")
     @ApiOperation(value = "Rotate DistroX secrets", produces = MediaType.APPLICATION_JSON, nickname = "rotateDistroXSecrets", hidden = true)
     FlowIdentifier rotateSecrets(@Valid @NotNull DistroXSecretRotationRequest request);
+
+    @POST
+    @Path("/crn/{crn}/refresh_datalake_config")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Refresh Data lake configuration", produces = MediaType.APPLICATION_JSON, nickname = "refreshDatalakeConfig")
+    FlowIdentifier refreshDatalakeConfig(@ValidCrn(resource = {CrnResourceDescriptor.DATAHUB}) @PathParam("crn") String crn);
```

Comment:
refresh_datalake_config means updating the datalake itself. Is this what we try to achieve here?

Reply:
@doktoric this is refreshing the stored DL configuration on the DH associated with the provided crn. Kind of confusing but not sure if it can be worded much better without being too long.